src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -583,32 +596,24 @@ export default {
 				? moment().add(1, 'weeks').day(1).hour(8)
 				: null
 
-			return [
-				{
-					key: 'laterToday',
-					timestamp: this.getTimestamp(laterTodayTime),
-					label: t('spreed', 'Later today – {timeLocale}', { timeLocale: laterTodayTime?.format('LT') }),
-					ariaLabel: t('spreed', 'Set reminder for later today'),
-				},
-				{
-					key: 'tomorrow',
-					timestamp: this.getTimestamp(tomorrowTime),
-					label: t('spreed', 'Tomorrow – {timeLocale}', { timeLocale: tomorrowTime?.format('ddd LT') }),
-					ariaLabel: t('spreed', 'Set reminder for tomorrow'),
-				},
-				{
-					key: 'thisWeekend',
-					timestamp: this.getTimestamp(thisWeekendTime),
-					label: t('spreed', 'This weekend – {timeLocale}', { timeLocale: thisWeekendTime?.format('ddd LT') }),
-					ariaLabel: t('spreed', 'Set reminder for this weekend'),
-				},
-				{
-					key: 'nextWeek',
-					timestamp: this.getTimestamp(nextWeekTime),
-					label: t('spreed', 'Next week – {timeLocale}', { timeLocale: nextWeekTime?.format('ddd LT') }),
-					ariaLabel: t('spreed', 'Set reminder for next week'),
-				},
-			].filter(option => option.timestamp !== null)
+			const times = {
+				'Later today': laterTodayTime,
+				Tomorrow: tomorrowTime,
+				'This weekend': thisWeekendTime,
+				'Next week': nextWeekTime,
+			}
+
+			return Object.entries(times).map(([key, moment]) => {
+				return {
+					key: key.toLowerCase().replace(' ', '_'),
+					timestamp: this.getTimestamp(moment),
+					label: t('spreed', '{readableTime} – {timeLocale}', {
+						readableTime: key,
+						timeLocale: moment?.format('LT'),
+					}),
+					ariaLabel: t('spreed', `Set reminder for ${key.toLowerCase()}`),
```

Comment:
This does not work for translations. Please revert to old pattern with fixed strings.

See https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html#important-notes for more information

Reply:
The aria label is translatable but I am not sure if the label is translatable. I think that's where the translation broke ? 
![image](https://github.com/nextcloud/spreed/assets/84044328/70312206-c41c-40f0-9d47-04e4945a5b26)

Anyway, for such cases, how to translate strings with variable part that should be translated ?

Reply:
> Anyway, for such cases, how to translate strings with variable part that should be translated ?

This is called string concatenation and does not work in all languages. Just don't use variabled strings. Go back to the predefined array that was here. (It's also unrelated to "Hide copy message button")

Reply:
Yes, it was just an attempt to reduce the redundancy